de-lib/src/main/java/org/iplantc/de/client/models/webhooks/WebhookType.java

```diff
@@ -0,0 +1,20 @@
+package org.iplantc.de.client.models.webhooks;
+
+import org.iplantc.de.client.models.HasId;
+
+/**
+ * Type of webhook. E.g: Slack, Zapier
+ *
+ * Created by sriram on 10/31/17.
+ */
+public interface WebhookType extends HasId {
+
+    String getType();
+
+    void setType(String type);
+
+    String getTemplate();
+
+    void setTemplate(String template);
```

Comment:
Is the user is able to view or update a webhook type's template?
If not, we could remove the template from this model and the endpoint, and it will save some bandwidth so these strings don't have to shuffle back and forth from the clients.

Reply:
Ya we can remove the template field from here and make this field optional on the services. 👍